docs/getting_started/basics/debug.md

```diff
@@ -0,0 +1,29 @@
+# Debug Workflows
+
+While working on a workflow, debugging may be necessary to ensure the graph execution produces the expected results. This page outlines Shift's features designed for workflow debugging.
+
+
+## Errored Nodes & Traceback
+When a node does not execute successfully it will turn red and the workflow execution will be interrupted. In such case, the node will store the traceback generated by the error. To inspect the traceback, `Right Click` on the node and select the *Traceback* option. This will pop up a dialog with the error traceback content. 
+
+![Breakpoint](../../images/traceback.gif)
+
+Once the root of the error is fixed, the node will turn blue and its traceback will be cleared when it gets recomputed. 
+
+### Logger
+The error traceback information can be consulted in the [Logger widget](../ui_overview/#logger) too. The logger also reports warnings raised by nodes, and other Shift messages such as plug connection errors or other general logs related to the workflow or the application itself. 

```

Comment:
```suggestion
### Logger

The error traceback information can be consulted in the [Logger widget](../ui_overview/#logger) too. The logger also reports warnings raised by nodes, and other Shift messages such as plug connection errors or other general logs related to the workflow or the application itself. 

```

Reply:
@albertochimentiinbibo Markdown does not care about empty lines after a title

Reply:
Done